CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+### Fixed
+- **CUMULUS-3116**
+  - Reverted the default ElasticSearch sorting behavior to the pre-13.3.0 configuration
+  - Results from ElasticSearch are sorted by default by the `timestamp` field. This means that the order
+  is not guaranteed if two or more records have identical timestamps as there is no secondary sort/tie-breaker.
```

Comment:
I wonder if this sort of note is helpful to add to the es-client package readme or something 🤔

Reply:
Sure, I think that makes sense to call out the default behavior: https://github.com/nasa/cumulus/pull/3161/commits/d8a662bbf6ed0a69f821c279b40852bd8f72fc95

Also, if users don't like it or it doesn't do what they expect, they can always update their API calls to sort differently.